packages/fxa-auth-server/lib/metrics/glean/server_events.ts

```diff
@@ -2107,8 +2247,8 @@ class EventsServerEventLogger {
     utm_term: string;
   }) {
     const event = {
-      category: 'third_party_auth_set_password',
-      name: 'complete',
+      category: 'third_party_auth',
+      name: 'set_password_complete',
```

Comment:
@akkomar Is this going to break something?

Reply:
This is fine.
This will be a new metric from telemetry ingestion point of view. FWIW on https://dictionary.telemetry.mozilla.org/apps/accounts_backend there is a `Show expired and removed metrics` checkbox - if selected, the old form of this event will show up.